categories/views.py

```diff
@@ -1,3 +1,43 @@
-from django.shortcuts import render
+import json, csv
+
+from django.http            import JsonResponse
+from django.views           import View
+
+from .models                import Category, SubCategory, CategoryJoin
+
+class CategoryView(View):
+  def get(self,request):
+    try:
+      subcategory = SubCategory.objects.all()
+      result = [i.name for i in subcategory]
+
+
+      return JsonResponse({'message': result}, status=200)
+    except:
+      return JsonResponse({'message': 'KEY_ERROR'}, status=400)
```

Comment:
1. get method 내에서 Key Error가 발생할 수 있는 경우가 없는데, Key Error 처리주고 있습니다. 불필요한 코드는 지워주세요.
2. 데이터를 반환해주실 때, 데이터를 대표하는 id 값도 같이 반환해주세요.!
```suggestion
    try:
      sub_categories = [{
          "id" : sub_category.id, 
          "name" : sub_category.name
      } for sub_category in SubCategory.objects.all()]
      
      return JsonResponse({'sub_categories': sub_categories}, status=200)
```

Reply:
확인했습니다!